audb/core/api.py

```diff
@@ -26,11 +26,13 @@
 def available(
     *,
     only_latest: bool = False,
+    repositories: Repository | Sequence[Repository] = None,
 ) -> pd.DataFrame:
     r"""List all databases that are available to the user.
 
     Args:
         only_latest: include only latest version of database
+        repositories: search only in the given repositories
```

Comment:
I would mention what happens when it is `None` in the docstring (falls back on the value of `config.REPOSITORIES`).

Reply:
I updated the docstring to

![image](https://github.com/user-attachments/assets/bdf6e10d-a9fd-4088-a18c-903cd9ce5f37)

Reply:
Nice. Resolving (hopefully only) this conversation.